pkg/controller/perconaservermongodb/smart.go

```diff
@@ -152,8 +152,9 @@ func (r *ReconcilePerconaServerMongoDB) smartUpdate(cr *api.PerconaServerMongoDB
 		}
 	}
 
-	log.Info("doing step down...")
-	err = mongo.StepDown(context.TODO(), client)
+	unsafe := cr.Spec.UnsafeConf && len(list.Items) == 1
+	log.Info(fmt.Sprintf("doing step down (force: %t)...", unsafe))
```

Comment:
No need to use `fmt` here

Reply:
do you recommend this format?

`log.Info("doing step down...", "force", forceStepDown)`

Reply:
Yes

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/651/commits/bc2a469c7302dfa68d26504037062857caed8f9b